src/commands/contract/instantiate.ts

```diff
@@ -48,7 +49,7 @@ export default class ContractInstantiate extends Command {
         lcd,
       });
 
-      admin = signer.key.accAddress;
+      admin = signer.key.accAddress('terra');
```

Comment:
can we add another flag (something like prefix) to terrain instead of hardcoding `terra` here? so we can use terrain on any cosmos chains

Reply:
if user wants to create app on injective, they can do `terrain new test_app --prefix inj`

Reply:
this makes terrain more flexible for future feather chains

Reply:
This will be coming in a future PR! 🔜